### PR TITLE
Getting Started > Installation docs

### PR DESCRIPTION
**Pull request by achambers**

### CODE REVIEW

File: docs/v0.5.x/installation.md [hunk 1]
```diff
@@ -2,8 +2,13 @@
...
 ---
 
-To install, run
+ember-cli-deploy is just an Ember CLI addon so it's super easy to install:
 
 ```
 ember install ember-cli-deploy
 ```
+
+But ember-cli-deploy doesn't work just on it's own. You also need to install plugins to actually do the deployment work.
```
Comment (lukemelia) [inline docs/v0.5.x/installation.md]: "But ember-cli-deploy doesn't work just on it's own." -> "But ember-cli-deploy doesn't do much by itself."

File: docs/v0.5.x/installation.md [hunk 2]
```diff
@@ -2,8 +2,13 @@
...
+
+Jump on over to the [Plugins Overview](./plugins-overview) to understand how plugins work. When you're all over that,
```
Comment (lukemelia) [inline docs/v0.5.x/installation.md]: "you're all over that" -> "you feel comfortable with the concept of a plugin"

File: docs/v0.5.x/installation.md [hunk 3]
```diff
@@ -2,8 +2,13 @@
...
+you can see the list of plugins that we currently support on the [Plugins Page](./plugins).
```
Comment (lukemelia) [inline docs/v0.5.x/installation.md]: "that we currently support" -> "that are currently available"

**Comment by achambers** [comment]
Thanks @lukemelia. Updated this one as well.

**Pull request merged**